lib/schema.ts

```diff
@@ -114,18 +121,18 @@ function buildFields(schema, rLevelParentMax, dLevelParentMax, path) {
               opts.fields,
               rLevelMax,
               dLevelMax,
-              path.concat([name]))
+              path.concat(name))
       };
-
-      if (opts.type == 'LIST' || opts.type == 'MAP') fieldList[name].originalType = opts.type;
-
```

Comment:
Removed for now because based on types file, maps and list are not being used

Reply:
I'm not sure here either. Because the thrift file does include them. @shannonwells / @enddynayn any idea?